<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#titleLabel {
background: white;
color: blue;
font-size: 20px;
border: none;
border-bottom:  1px solid black;
padding: 5px;
}

#mainFrame {
border: none;
background: white;
}

#loginFrame {
background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ddf, stop: 1 #aaf);
border: 1px solid gray;
padding: 10px;
border-radius: 25px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>The Battleship</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="mainPage">
          <widget class="QTextEdit" name="roomTextEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>331</y>
             <width>271</width>
             <height>171</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QListWidget" name="userListWidget">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>430</y>
             <width>256</width>
             <height>71</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="sayLineEdit">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>512</y>
             <width>211</width>
             <height>27</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="sendButton">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>510</y>
             <width>50</width>
             <height>27</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="serverLineEdit">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>10</y>
             <width>113</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>f6v</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="userLineEdit">
           <property name="geometry">
            <rect>
             <x>400</x>
             <y>10</y>
             <width>113</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>feri</string>
           </property>
          </widget>
          <widget class="QPushButton" name="connectButton">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>10</y>
             <width>99</width>
             <height>27</height>
            </rect>
           </property>
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>15</y>
             <width>68</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Hostname</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="portSpinBox">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>10</y>
             <width>71</width>
             <height>27</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1024</number>
           </property>
           <property name="maximum">
            <number>65535</number>
           </property>
           <property name="value">
            <number>8787</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>209</x>
             <y>14</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>324</x>
             <y>15</y>
             <width>81</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Your Name</string>
           </property>
          </widget>
          <widget class="QTableView" name="myField">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>250</width>
             <height>250</height>
            </rect>
           </property>
          </widget>
          <widget class="QLineEdit" name="targetLineEdit">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>530</x>
             <y>326</y>
             <width>113</width>
             <height>27</height>
            </rect>
           </property>
          </widget>
          <widget class="QTableView" name="enemyField">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>70</y>
             <width>250</width>
             <height>250</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>460</x>
             <y>330</y>
             <width>68</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Target</string>
           </property>
          </widget>
          <widget class="Line" name="line">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>37</y>
             <width>741</width>
             <height>16</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>410</y>
             <width>68</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Users</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>81</width>
             <height>17</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Your Field</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>390</x>
             <y>50</y>
             <width>101</width>
             <height>17</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Enemy's Field</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>50</y>
             <width>111</width>
             <height>20</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Shipblocks left:</string>
           </property>
          </widget>
          <widget class="QLabel" name="shipsLabel">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>50</y>
             <width>31</width>
             <height>17</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>(18)</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="arrangePage">
          <widget class="QTableView" name="initialField">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>260</y>
             <width>242</width>
             <height>247</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>220</y>
             <width>201</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Arrange Your Field</string>
           </property>
          </widget>
          <widget class="QPushButton" name="startButton">
           <property name="geometry">
            <rect>
             <x>450</x>
             <y>457</y>
             <width>110</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
          <widget class="QListWidget" name="shipsListWidget">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>280</y>
             <width>131</width>
             <height>161</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>30</y>
             <width>621</width>
             <height>151</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>The game is played on four grids, two for each player. The grids are 10x10 square and the individual squares in the grid are identified by letter and number.On one grid the player arranges ships and records the shots by the opponent. On the other grid the player records his/her own shots.
Before play begins, each player secretly arranges their ships on their primary grid. Each ship occupies a number of consecutive squares on the grid, arranged either horizontally or vertically. The number of squares for each ship is determined by the type of the ship. The ships cannot overlap. The types and numbers of ships allowed are the same for each player.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignJustify|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>91</width>
             <height>21</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
          <widget class="Line" name="line_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>190</y>
             <width>661</width>
             <height>21</height>
            </rect>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>256</y>
             <width>91</width>
             <height>17</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Hint:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>276</y>
             <width>131</width>
             <height>231</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>1. Select a ship.

2. Click on the grid to assign a position for the starting side.

3. Click on one of the ending sides.

4. Repeat 1-3 until all ships are arranged.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>440</x>
             <y>260</y>
             <width>68</width>
             <height>17</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Ships:</string>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>roomTextEdit</tabstop>
  <tabstop>userListWidget</tabstop>
  <tabstop>sayLineEdit</tabstop>
  <tabstop>sendButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sayLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
